delivery/app.py

```diff
@@ -39,15 +40,18 @@ def routes(**kwargs):
         url(r"/api/1.0/runfolders/(.+)/projects", ProjectsForRunfolderHandler,
             name="projects_for_runfolder", kwargs=kwargs),
 
-
         url(r"/api/1.0/stage/runfolder/(.+)", StagingRunfolderHandler,
             name="stage_runfolder", kwargs=kwargs),
+        url(r"/api/1.0/stage/project/(.+)", StageGeneralDirectoryHandler,
```

Comment:
I just meant that if you wanna follow semantic versioning then when you add something that breaks the interface, you bump arteria-delivery's version up to 2.0 from 1.0, and then preferably the routes should follow along.